go-runtime/compile/schema.go

```diff
@@ -398,6 +402,56 @@ func visitCallExpr(pctx *parseContext, node *ast.CallExpr, stack []ast.Node) {
 	}
 }
 
+// validateCallExpr validates all function calls
+// checks if the is directly:
+// - calling external verbs
+// - publishing to an external module's topic
+func validateCallExpr(pctx *parseContext, node *ast.CallExpr) {
```

Comment:
We do want this to be recursive at some point, such that if you call a function that calls a verb, we can track that.

Reply:
This should already be checking this case because we aren't just checking within functions defined as verbs:
```go
//ftl:verb
func Verb(...) (...) {
   helper()
   ...
}

func helper() {
    externalmodule.Verb() // <--- Error here
}

```